api/Element.json

```diff
@@ -6487,11 +6487,11 @@
               }
             ],
             "safari": {
-              "version_added": "6",
+              "version_added": "5.1",
```

Comment:
Is this 6&rarr;5.1 change based on testing something in both 5.1 and 6? Early support is tricky, there's both `webkitRequestFullscreen` and `webkitRequestFullScreen` which the data doesn't capture, and it's easy to get things wrong here I think.

Reply:
Yep, this came from testing for `webkitRequestFullscreen` (no capital S) specifically!